src/auto/injector.js

```diff
@@ -849,7 +853,7 @@ function createInjector(modulesToLoad, strictDi) {
       if (!isBoolean(result)) {
         // Workaround for MS Edge.
         // Check https://connect.microsoft.com/IE/Feedback/Details/2211653
```

Comment:
Maybe you could change the comment at the same time to:
```
// Support: Edge 12-13 only
// See https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/6156135/
```

It's fixed in the Edge preview and the bug has moved.